KomaMRICore/test/test_files/Utils.jl

```diff
@@ -0,0 +1,75 @@
+
+    # For EPI
+    FOVkx = Nx / FOVx                                       # The FOV of the k-space traversed by the ADC in the x-direction
+    Area_adc = FOVkx / γ                                    # The area of the rectangular gradient traversed by the ADC in the x-direction
+    Tgx = Nx*Δtadc                                          # The duration of the flat top of the x-gradient
+    Agx = Area_adc / Tgx                                    # The amplitude of the x-gradient
+    ζgx = ceil(Agx / Smax; digits=Δtgr_digits)              # The rise and fall time of the the x-gradient
+    Tadc = Tgx - Δtadc                                      # The duration of the ADC
+    ΔDadc = ζgx + Δtadc/2                                   # The delay of the ADC
+    Δky = 1 / FOVy                                          # The resolution of the k-space in the y-direction
+    Area_gy = Δky / γ                                       # The area of the triangular blip gradient traversed in the y-direction
+    ζgy = ceil(sqrt(Area_gy / Smax); digits=Δtgr_digits)    # The rise and fall time of the the triangular y-gradient (ζ = sqrt(Area/Slewrate) = sqrt((G*ζ)/(G/ζ)))
+    Agy = Area_gy / ζgy                                     # The amplitude of the triangular y-gradient
+
+    # For dephasers
+    Area_gx = Area_adc + Agx * ζgx                              # The total area of the x-gradient (Area_gx = ◢ + Area_adc + ◣ = Area_adc + ■ = Area_adc + Agx * ζgx)
+    Area_gxo = 1/2 * Area_gx                                    # The half-area of the x-gradient
+    ζgxo = ceil(sqrt(Area_gxo / Smax); digits=Δtgr_digits)      # The rise and fall time of the the triangular dephasing x-gradient (ζ = sqrt(Area/Slewrate) = sqrt((G*ζ)/(G/ζ)))
+    Agxo = Area_gxo / ζgxo                                      # The amplitude of the triangular dephasing x-gradient
+    Area_ky = Ny*Area_gy                                        # The width of the k-space traversed by the y-gradient in the y-direction
+    Area_gyo = 1/2 * Area_ky                                    # The half-width of the k-space in the y-direction
+    ζgyo = ceil(sqrt(Area_gyo / Smax); digits=Δtgr_digits)      # The rise and fall time of the the triangular dephasing y-gradient (ζ = sqrt(Area/Slewrate) = sqrt((G*ζ)/(G/ζ)))
+    Agyo = Area_gyo / ζgyo                                      # The amplitude of the triangular dephasing y-gradient
+
+    # Define the sequence
+    seq_excitation = Sequence([Grad(0.0, 0.0)], [RF(Arf, Trf)])
+    seq_dephaser = Sequence(reshape([Grad(-Agxo, 0.0, ζgxo); Grad(Agyo, 0.0, ζgyo)], :, 1))
+    seq_epi = Sequence(vcat(
+        [(i % 2 == 0) ? Grad(Agx*(-1)^(i÷2), Tgx, ζgx) : Grad(0.0, 0.0, ζgy) for i in 0:2*Ny-1],
+        [(i % 2 == 1) ? Grad(-Agy, 0.0, ζgy)           : Grad(0.0, Tgx, ζgx) for i in 0:2*Ny-1]))
+    seq_epi.ADC = [(i % 2 == 0) ? ADC(Nx, Tadc, ΔDadc) : ADC(0, 0.0) for i in 0:2*Ny-1]
```

Comment:
For ease of understanding I would do this in a for loop (`seq += ...`).

Reply:
Rename `seq_ex..`, `seq_..` to just the thing they represent and do `seq += ex` ...

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/7a87897111345002932a3a569178b48ae80afea8